relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -445,6 +445,15 @@ pub fn extract_tags(
                 }
             }
         }
+        if let Some(measurements) = span.measurements.value() {
+            if span_op.starts_with("ui.interaction") && measurements.contains_key("inp") {
```

Comment:
Note: the metric spec is defined for `ui.interaction.`:
```suggestion
            if span_op.starts_with("ui.interaction.") && measurements.contains_key("inp") {
```
It probably doesn't make much difference, but we should align these.